2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,17 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+const findSafeOxygenLevel = (oxygenArray) =>
```

Comment:
Great use of arrow function. You may use **endsWith** instead of checking array content